lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -269,7 +269,7 @@
     returned: success
     type: string
     sample: file
-status:
+status_code:
```

Comment:
LGTM. It doesn't look like `state` gets returned either.

Reply:
IIRC, I saw it return <code>state: "absent"</code> once when I got a "Connection reset by peer" error. I guess that is to say I'm not really sure what its relationship is to when it should actually be returned.